tools/cli/src/utils.rs

```diff
@@ -32,13 +33,36 @@ fn run_command<T: Into<Stdio>>(
     mut command: Command,
     stdout_config: T,
 ) -> Result<String, anyhow::Error> {
-    let process = command.stdout(stdout_config).spawn()?;
+    let process = command.stdout(stdout_config).spawn().map_err(|e| {
+        anyhow::anyhow!(
+            r#"cannot run "{}": {}"#,
```

Comment:
It worth using debug format for the command name.

Reply:
ditto